README.md

```diff
@@ -1 +1,43 @@
-# oop-school-library
\ No newline at end of file
+# oop-school-library
+This project is a school library store that allow to the librarian to record what books are in the library and who borrows them but also the system can allow to add new students or teachers, to add a new book and to save records of who borrowed a given book and when
+
+# Built With
+Major languages : Ruby
+Frameworks : none
+Technologies used : Git, Gitflows, Github, Rubocop and Rubygem
+# Getting Started
+To get a local copy up and running follow these simple example steps.
+
+# Prerequisites
+Have a computer and internet connection
+Have Ruby installed on your computer
+Hava a basic knowledge of Ruby and OOP concept
+Have visual-studio code or any other code editor installed on your computer.
+# Setup
+In order to get a copy of this project you need to download it from git@github.com:divinecharlotte/oop-school-library.git
+Extract the zipped file and open it in your code editor
+# Install
+Run the command bellow in your terminal to get all required files
+`bundle install`
+# Run tests
+you can run one of the followin command in your terminal
+
+> Rubocop --color
+
+
+> Rubocop -A
+
+## Authors
+
+👤 **Charlotte Divine**
+
+- GitHub: [divinecharlotte](https://github.com/divinecharlotte)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/charlotte-divine-dusenge/)
+
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the issues page.
+
+Show your support
+Give a star if you like this project!
```

Comment:
- Please, kindly ensure you've followed the layout format as used in the readme template [here](https://github.com/microverseinc/readme-template) to make your readme more professional.
- Add the License section.
- Update your issues page to link to the issues page of your project on Github.
- Add the Acknowledgement section also to make your readme more professional.